core/templates/dev/head/player/exploration_player.html

```diff
@@ -19,11 +19,13 @@
 {% endblock header_js %}
 
 {% block navbar_breadcrumb %}
-  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb" ng-controller="InformationCard">
     <li>
       <span class="oppia-navbar-breadcrumb-separator"></span>
       <span class="oppia-exploration-title protractor-test-exploration-header" itemprop="description">{{exploration_title}}</span>
     </li>
+    <li ng-click="showInformationCard()" tooltip="Information Card" tooltip-placement="bottom">
+    <img src="/third_party/static/material-design-icons-1.0.1/ic_info_white_48dp.png" class="oppia-navbar-breadcrumb-icon"></li>
```

Comment:
This should be indented by 2, and the `</li>` moved to the next line. Also, when hovering on this image, the mouse cursor should change to a pointer to indicate that it's clickable.